commcare_connect/opportunity/tables.py

```
@@ -844,10 +844,70 @@ def render_actions(self, record):
+
+    class Meta:
+        model = UserVisit
+        sequence = (
+            "date_time",
+            "entity_name",
+            "deliver_unit",
+            "payment_unit",
+            "last_activity",
+            "status",
+        )
+        fields = []
```

Ruff (0.15.9)

[warning] Mutable default value for class attribute

(RUF012)